src/107-Arduino-TMF8801.h

```diff
@@ -75,6 +77,8 @@ class ArduinoTMF8801 : public LengthSensorBase
   TMF8801::AlgoState const & _algo_state;
   unit::Length _distance;
 
+  uint8_t gpio_control;
```

Comment:
```suggestion
  uint8_t _gpio_control;
```
Typically member variables are prefixed with `_`.